firmware/spade/src/rpi/main.c

```diff
@@ -179,6 +183,24 @@ static int load_new_scripts(void) {
   return upl_stdin_read();
 }
 
+void mount_sd_card() {
+  yell("mounting SD card");
```

Comment:
nit (forward-looking) - I am mentally equating "yell" with higher log level - something like the mounting of hardware should probably be at debug/verbose log level.  My suggestion is, given the scale of where we're at, low vs high might be all the resolution we need.

Reply:
I've sorta just been using `yell` for debugging instead of for high log level - my concern with establishing `yell` as just high-level logging is that as an alias of `puts`, it only takes a `char *`, and no builtin formatting, meaning that if I wanted to embed a value into a high-level log, I would have to use something like `snprintf` and then log. `printf` can do it quick in one line.

I would propose that to communicate log level, we can just put the level in the log content itself, like by prefixing the log with `(HIGH)` or `(LOW)`. Can be implemented along with other logging cleanup in a future PR.